app/src/main/java/org/fossasia/susi/ai/data/device/RoomModel.kt

```diff
@@ -0,0 +1,28 @@
+package org.fossasia.susi.ai.data.device
+
+import org.fossasia.susi.ai.data.contract.IRoomModel
+import org.fossasia.susi.ai.dataclasses.AddDeviceQuery
+import org.fossasia.susi.ai.rest.ClientBuilder
+import org.fossasia.susi.ai.rest.responses.susi.GetAddDeviceResponse
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import timber.log.Timber
+
+class RoomModel : IRoomModel {
+    private lateinit var addDeviceResponseCall: Call<GetAddDeviceResponse>
```

Comment:
Why does this need to be a class member?

Reply:
This is as per the architecture or the way it has been implemented in other files. So, I created it. Also, we will have more functions related to room in the roomModel class.